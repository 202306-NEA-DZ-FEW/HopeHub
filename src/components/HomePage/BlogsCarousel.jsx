import { collection, getDocs, limit, orderBy, query } from "firebase/firestore";
import Image from "next/image";
import Link from "next/link";
import { useTranslation } from "next-i18next";
import React, { useEffect, useState } from "react";
import { TfiAngleLeft, TfiAngleRight } from "react-icons";
import Slider from "react-slick";

import "slick-carousel/slick/slick.css";
import "slick-carousel/slick/slick-theme.css";

import { db } from "../../util/firebase";

const BlogsCarousel = () => {
    const { t } = useTranslation("common");

    const [blogs, setBlogs] = useState([]); // State to store blog data

    // Function to fetch blog data from Firestore
    const fetchBlogsFromFirestore = async () => {
        const blogsCollection = collection(db, "blogs");
        const q = query(blogsCollection, orderBy("date", "desc"), limit(6)); // Sort by date in descending order and limit to the last 6 entries
        const data = await getDocs(q);
        const blogData = [];
        data.forEach((doc) => {
            const blog = doc.data();
            blogData.push(blog);
        });
        setBlogs(blogData);
    };

    useEffect(() => {
        fetchBlogsFromFirestore();
    }, []); // Fetch data when the component mounts

    const CustomPrevArrow = ({ onClick }) => (
        <div className='custom-arrow-l' onClick={onClick}>
            <TfiAngleLeft size={30} color='black' />
        </div>
    );

    const CustomNextArrow = ({ onClick }) => (
        <div className='custom-arrow-r' onClick={onClick}>
            <TfiAngleRight size={30} color='black' />
        </div>
    );

    const settings = {
        infinite: true,
        speed: 500,
        slidesToShow: 3,
        slidesToScroll: 1,
        prevArrow: <CustomPrevArrow />,
        nextArrow: <CustomNextArrow />,
        responsive: [
            {
                breakpoint: 1024,
                settings: {
                    slidesToShow: 2,
                    slidesToScroll: 1,
                    infinite: true,
                },
            },
            {
                breakpoint: 768,
                settings: {
                    slidesToShow: 1,
                    slidesToScroll: 1,
                    initialSlide: 0,
                },
            },
        ],
    };

    return (
        <div className='bg-Primary dark:bg-Dark_Primary pb-16 w-full'>
            <h1 className='mx-6 mt-4 mb-6 text-base md:mb-4 md:text-3xl md:mx-9 md:mt-10 font-poppins uppercase font-medium inline-block dark:text-NeutralWhite text-NeutralBlack'>
                {t("RECENT BLOGS")}
            </h1>
            <div className='px-20'>
                <Slider {...settings}>
                    {blogs.map((blog) => (
                        <Link key={blog.id} href={`/blogs/${blog.id}`}>
                            <div
                                key={blog.id}
                                className='relative px-2 md:mx-4 lg:mx-2 2xl:mx-10'
                            >
                                <Image
                                    src={blog.imageURL}
                                    alt='therapists'
                                    width={500}
                                    height={60}
                                    layout='fixed'
                                    className='filter brightness-50'
                                />
                                <div className='absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4'>
                                    <h1 className='text-Primary dark:text-Dark_NeutralWhite text-base text-center md:text-2xl font-normal font-poppins'>
                                        {blog.content.title}
                                    </h1>
                                    <h1 className='text-white text-xs text-center md:text-sm font-normal font-poppins'>
                                        {blog.subTitle}
                                    </h1>
                                </div>
                            </div>
                        </Link>
                    ))}
                </Slider>
            </div>
        </div>
    );
};

export default BlogsCarousel;
