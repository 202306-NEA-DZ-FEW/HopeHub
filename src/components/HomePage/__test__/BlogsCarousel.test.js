import renderer from "react-test-renderer";
import "./matchMedia.mock";

import BlogsCarousel from "../BlogsCarousel";

it("renders correctly", () => {
    const tree = renderer.create(<BlogsCarousel />).toJSON();
    expect(tree).toMatchSnapshot();
});
