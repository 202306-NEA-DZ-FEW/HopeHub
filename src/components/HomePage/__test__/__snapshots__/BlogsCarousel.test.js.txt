// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-Primary pb-9 w-full"
>
  <h1
    className="mx-6 mt-4 mb-6 text-base md:mb-4 md:text-3xl md:mx-9 md:mt-10 font-poppins uppercase font-black inline-block"
  >
    RECENT BLOGS
  </h1>
  <div
    className="px-20"
  >
    <div
      className="slick-slider slick-initialized"
      dir="ltr"
    >
      <div
        className="custom-arrow-l"
        onClick={[Function]}
      >
        <svg
          color="black"
          fill="currentColor"
          height={30}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": "black",
            }
          }
          version="1.1"
          viewBox="0 0 17 17"
          width={30}
          xmlns="http://www.w3.org/2000/svg"
        >
          <g />
          <path
            d="M5.207 8.471l7.146 7.147-0.707 0.707-7.853-7.854 7.854-7.853 0.707 0.707-7.147 7.146z"
          />
        </svg>
      </div>
      <div
        className="slick-list"
        onClick={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={null}
        onMouseMove={null}
        onMouseUp={[Function]}
        onTouchCancel={null}
        onTouchEnd={[Function]}
        onTouchMove={null}
        onTouchStart={[Function]}
        style={Object {}}
      >
        <div
          className="slick-track"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          style={
            Object {
              "WebkitTransform": "translate3d(0px, 0px, 0px)",
              "WebkitTransition": "",
              "msTransform": "translateX(0px)",
              "opacity": 1,
              "transform": "translate3d(0px, 0px, 0px)",
              "transition": "",
            }
          }
        >
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={-3}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      The Work Life Balance
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Understanding how therapy helps!
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={-2}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Self Isolations
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Why we isolate when we need the most help.
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={-1}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Heal From Trauma
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Understanding Different Therapy Approaches
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={false}
            className="slick-slide slick-active slick-current"
            data-index={0}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Couples Therapy
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      How can it help your family?
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={false}
            className="slick-slide slick-active"
            data-index={1}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Your First Therapy Session
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      What should you talk about?
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={false}
            className="slick-slide slick-active"
            data-index={2}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Meditation
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      How it helped me deal with depression.
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={3}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      The Work Life Balance
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Understanding how therapy helps!
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={4}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Self Isolations
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Why we isolate when we need the most help.
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={5}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Heal From Trauma
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Understanding Different Therapy Approaches
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={6}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Couples Therapy
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      How can it help your family?
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={7}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Your First Therapy Session
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      What should you talk about?
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={8}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Meditation
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      How it helped me deal with depression.
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={9}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      The Work Life Balance
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Understanding how therapy helps!
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={10}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Self Isolations
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Why we isolate when we need the most help.
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={11}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <a
                href="/blogs/undefined"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="relative px-2 md:mx-4 lg:mx-2 2xl:mx-10"
                >
                  <img
                    alt="therapists"
                    className="filter brightness-50"
                    data-nimg="1"
                    decoding="async"
                    height={60}
                    loading="lazy"
                    onError={[Function]}
                    onLoad={[Function]}
                    src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
                    srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
                    style={
                      Object {
                        "color": "transparent",
                      }
                    }
                    width={500}
                  />
                  <div
                    className="absolute inset-0 flex flex-col items-center justify-center px-6 md:px-16 lg:px-4"
                  >
                    <h1
                      className="text-Primary text-base text-center md:text-2xl font-extrabold font-poppins"
                    >
                      Heal From Trauma
                    </h1>
                    <h1
                      className="text-white text-xs text-center md:text-sm font-bold font-poppins"
                    >
                      Understanding Different Therapy Approaches
                    </h1>
                  </div>
                </div>
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="custom-arrow-r"
        onClick={[Function]}
      >
        <svg
          color="black"
          fill="currentColor"
          height={30}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": "black",
            }
          }
          version="1.1"
          viewBox="0 0 17 17"
          width={30}
          xmlns="http://www.w3.org/2000/svg"
        >
          <g />
          <path
            d="M13.207 8.472l-7.854 7.854-0.707-0.707 7.146-7.146-7.146-7.148 0.707-0.707 7.854 7.854z"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
